.lando.yml

```diff
@@ -5,33 +5,55 @@ recipe: wordpress
 config:
   php: '7.2'
   via: nginx
-  webroot: trunk
+  webroot: public
   database: mariadb
   xdebug: false
 
+  conf:
+    server: config/wordpress.conf
+
 services:
   appserver:
     composer:
+      # Note installing phpunit should be temporary because core should really install it via composer on its own.
       phpunit/phpunit: "^6"
     install_dependencies_as_root:
       - apt-get update -y
       - apt-get install gnupg -y
-      - curl -sL https://deb.nodesource.com/setup_8.x | bash -
-      - apt-get install subversion -y
+      - curl -sL https://deb.nodesource.com/setup_10.x | bash -
       - apt-get install nodejs -y
-    install_dependencies_as_me_internal:
-      - mkdir ~/.npm-global
-      - npm config set prefix '~/.npm-global'
-      - npm install -g grunt-cli
-      - npm install -g gulp-cli
+      - apt-get install zip -y
+      - apt-get install subversion -y
+      # Allow loopback requests to work.
+      - echo "127.0.0.1  wordpressdev.lando.site" >> /etc/hosts
+    install_dependencies_as_me:
+      # Setup environment variables.
+      - if [ ! -e $LANDO_MOUNT/.env ]; then cp $LANDO_MOUNT/config/.env $LANDO_MOUNT/.env && sed -i "s#/app#$LANDO_MOUNT#" $LANDO_MOUNT/.env; fi
+    run:
+      # Setup WordPress for contributing via git.
+      - if [ ! -e $LANDO_MOUNT/public/trunk-git ]; then git clone --depth 1 https://github.com/WordPress/wordpress-develop.git $LANDO_MOUNT/public/trunk-git; fi
+      - if [ ! -e $LANDO_MOUNT/public/trunk-git/wp-config.php ]; then cp $LANDO_MOUNT/config/wp-config.php $LANDO_MOUNT/public/trunk-git/wp-config.php; fi
+      - if [ ! -e $LANDO_MOUNT/public/trunk-git/wp-tests-config.php ]; then cp $LANDO_MOUNT/config/wp-tests-config.php $LANDO_MOUNT/public/trunk-git/wp-tests-config.php; fi
+      - cd $LANDO_MOUNT/public/trunk-git && composer install
+      - cd $LANDO_MOUNT/public/trunk-git; npm install && npx grunt
+      # Setup WordPress for contributing via SVN.
+      - if [ ! -e $LANDO_MOUNT/public/trunk-svn ]; then svn co https://develop.svn.wordpress.org/trunk/ $LANDO_MOUNT/public/trunk-svn; fi
+      - if [ ! -e $LANDO_MOUNT/public/trunk-svn/wp-config.php ]; then cp $LANDO_MOUNT/config/wp-config.php $LANDO_MOUNT/public/trunk-svn/wp-config.php; fi
+      - if [ ! -e $LANDO_MOUNT/public/trunk-svn/wp-tests-config.php ]; then cp $LANDO_MOUNT/config/wp-tests-config.php $LANDO_MOUNT/public/trunk-svn/wp-tests-config.php; fi
+      - cd $LANDO_MOUNT/public/trunk-svn && composer install
+      - cd $LANDO_MOUNT/public/trunk-svn; npm install && npx grunt
```

Comment:
As opposed to doing `npm install` and `npx grunt` would it not be better to just copy what was done above for Git? Though for me, I would probably comment-out all of this on my own system since I'd checkout SVN inside of Git.

Reply:
The environments _might_ be different, so I'd rather not rely on copying for now. But as said before, I'm open to a later PR to leverage the combination of git and SVN as you suggested.

Something else I just noticed I'd like to fix here is that `composer install`, `npm install` and `npx grunt` should all happen conditionally IMO, to not require that on every `lando start`. We could check for existence of `vendor` / `node_modules` / `build` directories.

Reply:
Agreed. Short-circuiting installation and building makes rebuilding Lando so much faster.